doc/source/example-jax-from-centralized-to-federated.rst

```diff
@@ -0,0 +1,284 @@
+So what we really do here is we tell Flower through our :code:`NumPyClient` subclass which of our already defined functions to call for training and evaluation.
+We included type annotations to give you a better understanding of the data types that get passed around.
+
+.. code-block:: python
+
+
+    class MNISTClient(fl.client.NumPyClient):
+        """Flower client implementing MNIST classification using MXNet."""
+
+        def __init__(
+            self,
+            params: Dict,
+            grad_fn: Callable,
+            train_x: List[np.ndarray],
+            train_y: List[np.ndarray],
+            test_x: List[np.ndarray],
+            test_y: List[np.ndarray],
+        ) -> None:
+            self.params= params
+            self.grad_fn = grad_fn
+            self.train_x = train_x
+            self. train_y = train_y
+            self.test_x = test_x
+            self.test_y = test_y
+
+        def get_parameters(self) -> Dict:
+            # Return model parameters as a list of NumPy ndarrays,
+            parameter_value = []
+            for _, val in self.params.items():
+                parameter_value.append(np.array(val))
+            return parameter_value
+        
+        def set_parameters(self, parameters: List[np.ndarray]) -> Dict:
+            # Collect model parameters and set new weight values
+            value=jnp.ndarray
+            params_item = list(zip(self.params.keys(),parameters))
+            for item in params_item:
+                key = item[0]
+                value = item[1]
+                self.params[key] = value
+            return self.params
+
+        
+        def fit(
+            self, parameters: List[np.ndarray], config: Dict
+        ) -> Tuple[List[np.ndarray], int, Dict]:
+            # Set model parameters, train model, return updated model parameters
+            print("Start local training")
+            self.params = self.set_parameters(parameters)
+            self.params, loss, num_examples = jax_training.train(self.params, self.grad_fn, self.train_x, self.train_y)
+            results = {"loss": float(loss)}
+            print("Training results", results)
+            return self.get_parameters(), num_examples, results
+
+        def evaluate(
+            self, parameters: List[np.ndarray], config: Dict
+        ) -> Tuple[float, int, Dict]:
+            # Set model parameters, evaluate model on local test dataset, return result
+            print("Start evaluation")
+            self.params = self.set_parameters(parameters)
+            loss, num_examples = jax_training.evaluation(self.params,self.grad_fn, self.test_x, self.test_y)
+            print("Evaluation accuracy & loss", loss)
+            return (
+                float(loss),
+                num_examples,
+                {"loss": float(loss)},
+            )
+
+Having defined data loading, model architecture, training, and evaluation we can put everything together and train our :code:`Linear Regression` model.
```

Comment:
Isn't this the same sentence as above? Should we mention that it's now federated?